src/App.jsx

```diff
@@ -27,46 +27,90 @@ function App() {
     });
   }
 
+  function getOppDirection(dir) {
+    if (dir == DIRECTION.UP) {
```

Comment:
- Name functions & arg properly, change getOppDirection to getOppositeDirection and dir to direction.
- Can use a switch statement here.